util/config/make_sys_basic_types.py

```diff
@@ -200,22 +200,23 @@ def get_sys_c_types(docs=False):
         if chpl_type == 'c_ptr':
             handled_c_ptr = True
 
-    # Finally, print out set of asserts for module. They assert that the
-    # sizeof(<extern chpl type>) matches the sizeof(<chpl type>). E.g.
-    #
-    #   assert(sizeof(c_int) == sizeof(int(32)));
-    #
-    sys_c_types.append("""
-{
-  extern proc sizeof(type t): c_size_t;
-""")
-    for i, max_value in enumerate(max_values):
-        _, chpl_type, _ = _types[i]
-        if chpl_type.startswith('c_'):
-            chpl_value = _max_value_to_chpl_type.get(str(max_value))
-            sys_c_types.append('  assert(sizeof({chpl_type}) == sizeof({chpl_value}))'
-                               ';'.format(**locals()))
-    sys_c_types.append('}')
+    if not minimal:
```

Comment:
I'm okay with this, but I'd like to understand the motivation (and it might be useful to put that motivation in a comment on the `if` statement for posterity :) )

Reply:
The motivation is that `==` is not provided in minimal modules, which makes this code not be valid. I felt like this switch was more elegant than rolling a custom `==` as part of CTypes.

Reply:
Huh.  Good to know!